patches/chrome-test-BUILD.gn.patch

```diff
@@ -10,3 +10,11 @@ index bbeda5c83d65a8280e3c1114793966b092a27929..263becedb204b5bd07796c8c82dceaa3
      # Runtime dependencies
      data_deps = [
        "//chrome:browser_tests_pak",
+@@ -4215,6 +4216,7 @@ test("unit_tests") {
+     "//ui/web_dialogs:web_dialogs_unittests",
+     "//v8",
+   ]
++  deps += [ "//brave/test:brave_test_support_unit", ]
```

Comment:
what is this for?

Reply:
This is to fix a linking error that happens because of the `push_messaging_service_unittest.cc` having a dependency on `BraveTestingProfile` (via a chromium_src override) that needs to be satisfied.

Without this bit, the following error happens at link time:
```
ld.lld: error: undefined symbol: BraveTestingProfile::BraveTestingProfile()
>>> referenced by push_messaging_service_unittest.cc:73 (/home/mario/work/brave-browser/src/brave/chromium_src/chrome/browser/push_messaging/../../../../../chrome/browser/push_messaging/push_messaging_service_unittest.cc:73)
>>>               obj/chrome/test/unit_tests/push_messaging_service_unittest.o:((anonymous namespace)::PushMessagingTestingProfile::PushMessagingTestingProfile())
>>> did you mean: BraveTestingProfile::~BraveTestingProfile()
>>> defined in: obj/chrome/test/unit_tests/push_messaging_service_unittest.o
```

Reply:
I think maybe it's best to insert brave_testing_profile into the testing_profile .h/.cc through chromium_src overrides and then they'll be available here without any patches

Reply:
I didn't realize I could do that, thanks for the suggestion.

PR updated!